server/application-server/src/main/java/de/tum/in/www1/hephaestus/codereview/comment/IssueCommentConverter.java

```diff
@@ -17,4 +23,15 @@ public IssueComment convert(@NonNull GHIssueComment source) {
         return comment;
     }
 
+    @Override
+    public IssueComment update(@NonNull GHIssueComment source, @NonNull IssueComment comment) {
```

Comment:
I think the way to go is always using `update` in `convert` and moving the code from `convert` into `update`.

Reply:
This applies to all converters